modules/types/include/hephaestus/types/type_formatting.h

```diff
@@ -0,0 +1,45 @@
+//=================================================================================================
+// Copyright (C) 2024 FILICS GMBH
```

Comment:
guess this should be hephaestus contributors?

Reply:
Great find, updated.